--- pulseaudio-0.9.14.orig/src/modules/rtp/module-rtp-recv.c	2009-01-12 15:10:34.000000000 -0800
+++ pulseaudio-0.9.14/src/modules/rtp/module-rtp-recv.c	2009-02-03 17:53:31.000000000 -0800
@@ -453,7 +462,8 @@
     data.module = u->module;
     pa_sink_input_new_data_set_sample_spec(&data, &sdp_info->sample_spec);
 
-    s->sink_input = pa_sink_input_new(u->module->core, &data, 0);
+    s->sink_input = pa_sink_input_new(u->module->core, &data,
+        PA_SINK_INPUT_VARIABLE_RATE);
     pa_sink_input_new_data_done(&data);
 
     if (!s->sink_input) {
